### ISSUE TITLE
Possibly allow custom folder ordering, not a strict A-Z

### ISSUE THREAD

**Issue by shmup** [a85/POSTMan-Chrome-Extension#2424]
TL;DR - folders are nice, but for test runner ordering, I want them in: C, A, B order, but am forced to use A, B, C because of A-Z restriction. Possibly allow us to drag and reorder, as we can with individual requests.

---

We're going to have a few hundred tests running through newman that are built through Postman, and these will all be grouped into various collections.

But within the collection itself, I might be testing one _module_ that has 50 or so requests. These requests can also be categorized to a smaller level, and I'd like to split them up among folder Baz, Foo, Bar, etc. In part because of organization, and another part: I can only ever see about 30 requests at a time in the collections pane and have to scroll often. I'd like to just expand/collapse folders.

The problem with splitting them up into folders is that I lose the required order for newman and/or the test runner, because directories are forced into an Bar, Foo, Baz (A-Z) naming order.

**Comment by shmup** [comment]
I suppose the workaround right now is to just throw a number in front of the folder name. This is a fine enough solution I'm not even upset anymore, haha.

**Comment by sdnts** [comment]
@shmup This is actually in our immediate roadmap, and we're working on this. I'll let you know here when it is out :)

**Comment by abhijitkane** [comment]
@shmup This is available in the latest versions of Postman.
